backend/server.js

```diff
@@ -83,6 +100,13 @@ app.post('/updateUserAccountInfo', function(req, res) {
   updateUserProfileToDb(uri, req.body).then(response => {console.log(response); res.send(response)});
 });
 
+app.post('/verifyUserAccountInfo', function(req, res) {
```

Comment:
Yes, it seems like using POST is the way to go: https://stackoverflow.com/questions/5868786/what-method-should-i-use-for-a-login-authentication-request